README.md

```diff
@@ -8,34 +8,71 @@ This is studies project for Data Security course where project topic is blockcha
 
 Blockchain is a digital public ledger that records transactions. It is a distributed ledger that can be queried and verified. It is a public record of past transactions.
 
-### How blockchain works?
+### How blockchain works
 
 Blockchain works based on hash function and proof of work. Hash function is used to generate hash value and proof of work is used to verify hash value.
 
-### What is hash function?
+### What is hash function
 
-Hash function is used to generate hash value. Hash function is used to generate hash value.
+Hash function is used to generate hash value.
 
-### What is proof of work?
+### What is proof of work
 
-Proof of work is used to verify hash value. Proof of work is used to verify hash value.
+Proof of work is used to verify hash value
 
-### What is 51% attack?
+### What is 51% attack
 
 51% attack is a type of attack in which a computer program or a networked system is able to modify the data in a database or a file without the user's knowledge.
 
-## How to run this project?
+## How to run this project
 
 To run this project you need to install pip and then run:
 
 ```bash
-pip install -r requirements.txt
+python -m pip install -r requirements.txt
 ```
 
 After that to run project simply call:
 
 ```bash
-python3 /app/GUI/gui.py
+python ./app/GUI/gui.py
 ```
 
 And window with UI should be opend displaying staring block as well as buttions to start mining and stop mining and clear button.
+
+## How program works
+
+Starting screen is displayed with starting block.
+This is shown in UI screen below.
+
+![image](.github/images/start.png)
+
+Working progress is shown in UI screen below, as miner works blocks are added and displayed. If invalid (FAKE) block it is shown in red color, valid (GENUINE) block are in green, and any conflicting (CONFLICT) blocks (conflict means that there are blockchain of the same length) is shown in orange.
```

Comment:
1. There is no point in using "if" at the beginning of the second sentence.
2.  Remove _(FAKE)_, _(GENUINE)_ and _(CONFLICT)_ from here. The color of the block in GUI is based only on proof of work, not on the `data` field.